std/experimental/.gitignore

```diff
@@ -0,0 +1,7 @@
+
+*.o
+perpetual
+core
+Makefile
+perpetual.html
+
```

Comment:
Is this file needed once it would be in Phobos? All the other modules are fine without this.